server/sonar-web/src/main/js/apps/project-admin/links/Links.js

```diff
@@ -34,26 +33,17 @@ class Links extends React.PureComponent {
     links: PropTypes.array
   };
 
-  componentWillMount() {
-    this.handleCreateLink = this.handleCreateLink.bind(this);
-    this.handleDeleteLink = this.handleDeleteLink.bind(this);
-  }
-
   componentDidMount() {
     this.props.fetchProjectLinks(this.props.component.key);
   }
 
-  handleCreateLink(name, url) {
+  handleCreateLink = (name, url) => {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove curly braces and "return" from this arrow function body. ([javascript:S3524](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fstas%2Fless-backbone-more-react&issues=AWG-K9A8_Zsgh39atBHp&open=AWG-K9A8_Zsgh39atBHp)

[ISSUE_KEY]: AWG-K9A8_Zsgh39atBHp